tendermint/src/amino_types/block_id.rs

```diff
@@ -1,4 +1,4 @@
-use super::validate::{ConsensusMessage, ValidationError, ValidationErrorKind::*};
+use super::validate::{ConsensusMessage, Kind::*};
```

Comment:
Why are we sticking with the wild-card import, can we qualify what we exactly need?